src/stories/Library/shadow/Shadow.tsx

```diff
@@ -0,0 +1,25 @@
+const shadowValues = [
+  "shadow-low",
+  "shadow-medium",
+  "shadow-high",
+  "shadow-low-hover",
+  "shadow-medium-hover",
+  "shadow-high-hover",
+];
+
+export const Shadow = () => {
```

Comment:
Okay I know this is really nit-picky, but would it maybe make more sense to call this whole thing Shadows instead of Shadow? The Color variables file is also named in plural to reflect that there are more. 🙈 (also in the stories file Library / Shadows)

Reply:
No problem. I initially could not decide if it should be plural or not, and then looked at Tag and decided it should not. But i'm all for consistency, so I have renamed 👍